docs/admin/authorization/rbac.md

```diff
@@ -122,26 +132,27 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: read-secrets-global
 subjects:
-  - kind: Group # May be "User", "Group" or "ServiceAccount"
-    name: manager
+- kind: Group
+  name: manager
+  apiGroup: rbac.authorization.k8s.io
 roleRef:
   kind: ClusterRole
   name: secret-reader
   apiGroup: rbac.authorization.k8s.io
 ```
 
-## Referring to Resources
+### Referring to Resources
 
 Most resources are represented by a string representation of their name, such as "pods", just as it
 appears in the URL for the relevant API endpoint. However, some Kubernetes APIs involve a
-"subresource" such as the logs for a pod. The URL for the pods logs endpoint is:
+"subresource", such as the logs for a pod. The URL for the pods logs endpoint is:
 
 ```
 GET /api/v1/namespaces/{namespace}/pods/{name}/log
 ```
 
 In this case, "pods" is the namespaced resource, and "log" is a subresource of pods. To represent
-this in an RBAC role, use a slash to delimit the resource and subresource names. To allow a subject
+this in an RBAC role, use a slash to delimit the resource and subresource. To allow a subject
 to read both pods and pod logs, you would write:
```

Comment:
- [ ] This is the first use of "you" in the document.  
  While it reads acceptably, you could omit it:
> both pods and pod logs, use the following set of resources:

Alternately,
> both pods and pod logs, write: